src/detect.c

```diff
@@ -1631,7 +1728,7 @@ static void DetectRunFrames(ThreadVars *tv, DetectEngineCtx *de_ctx, DetectEngin
                     det_ctx->frame_id = frame->id;
                     SCLogDebug(
                             "%p/%" PRIi64 " sig %u (%u) matched", frame, frame->id, s->id, s->num);
-                    PacketAlertAppend(det_ctx, s, p, 0, alert_flags); // TODO tx id frame field
```

Comment:
Is this TODO about tx_id frame field still important? mentioning so it won't get lost...

Reply:
ya I think so

Reply:
Can it be made into a ticket?

Reply:
left a comment around the same place, for now